config.go

```diff
@@ -512,6 +513,14 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
+	if cfg.TBOpts.SpreadTicketPurchases {
+		str := "ticketbuyer.spreadticketpurchases option has been replaced by " +
+			"ticketbuyer.nospreadticketpurchases -- please update your config"
+		err := fmt.Errorf(str)
+		fmt.Fprintln(os.Stderr, err)
+		return loadConfigError(err)
```

Comment:
we don't error out for deprecated options, we warn and gracefully handle them.  In this case we should just log if it is set.